### PR TITLE
Slovak translations and minor bug fixes

### PR DESCRIPTION
**Pull request by jozefcipa**
#### Description of what you did:
Since I'm working on the project for a Slovak client I needed to implement local translations and decided to share it also with the community :)
It also fixes some minor bugs that I found during the translation process.

#### My PR is a:

- [ ] 💥 Breaking change
- [x] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [x] Admin
- [ ] Documentation
- [ ] Framework
- [x] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by jozefcipa** [comment]
cc: @profispojka, https://github.com/MartinN3 I've noticed you guys were involved in the Czech translation so I'd like you to have a look at Slovak's version as well (since our nations are as close as brothers 🇸🇰🇨🇿❤️), if you have time 😇

**Comment by profispojka** [comment]
> cc: @profispojka, @MartinN3 I've noticed you guys were involved in the Czech translation so I'd like you to have a look at Slovak's version as well (since our nations are as close as brothers 🇸🇰🇨🇿❤️), if you have time 😇

Hi @jozefcipa, I have checked your translation very quickly and found a few typos.

`"PopUpForm.Providers.linkedin2.providerConfig.redirectURL": "URL presmerovania, ktorú je potrebné pridať v nastaveniach aplikácieLinkedin",` - there should be space before "Linkedin"

`"PopUpForm.Providers.facebook.providerConfig.redirectURL": "URL presmerovania, ktorú je potrebné pridať v nastaveniach aplikácie Face ook",` - you are missing "b" in Facebook

And thats all. But thanks to you I have realized a few problems with our czech translation, so thank you :)

**Comment by jozefcipa** [comment]
@profispojka fixed, thanks

**Comment by profispojka** [comment]
@jozefcipa found untranslated string - form.attribute.component.option.repeatable.description

**Comment by jozefcipa** [comment]
@profispojka yes, I wasn't able to do a proper translation with examples because I don't really understand the meaning of this repeatable component, i.e. what's the difference between Repeatable component and Simple component

File: examples/getstarted/config/locales/sk_sk.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (soupette) [inline examples/getstarted/config/locales/sk_sk.json]: This file is not really needed as it is not used in the example app. Do you mind removing it?

File: packages/strapi-plugin-content-type-builder/admin/src/translations/sk.json [hunk 2]
```diff
@@ -0,0 +1,154 @@
+{
+  "attribute.boolean": "Boolean",
```
Comment (soupette) [inline packages/strapi-plugin-content-type-builder/admin/src/translations/sk.json]: Do you mind removing the "not translated keys" there's a fallback to English. This way we can automatically create issues on GitHub asking people from the community to fill the missing translations?

Thanks a lot

File: packages/strapi-plugin-upload/admin/src/containers/HomePage/saga.js [hunk 3]
```diff
@@ -65,12 +65,12 @@ function* uploadFiles(action) {
 
     if (newFiles.length > 1) {
       strapi.notification.success({
-        id: 'upload.notification.dropFile.success',
+        id: 'upload.notification.dropFiles.success',
```
Comment (soupette) [inline packages/strapi-plugin-upload/admin/src/containers/HomePage/saga.js]: Good catch! Thanks for fixing this one!

File: packages/strapi-admin/admin/src/components/LeftMenuLinkContainer/index.js [hunk 4]
```diff
@@ -43,7 +43,9 @@ function LeftMenuLinkContainer({ plugins, ...rest }) {
 
     return (
       <div key={j}>
-        <p className="title">{pluginsSections[current].name}</p>
+        <p className="title">
```
Comment (soupette) [inline packages/strapi-admin/admin/src/components/LeftMenuLinkContainer/index.js]: Do you mind changing it to so we know that the DOM stays unchanged:

```
<p className="title">
  <FormattedMessage {...messaged.contentTypes}>
    {title => title}
  </FormattedMessage>
</p>
```

Comment (jozefcipa) [reply packages/strapi-admin/admin/src/components/LeftMenuLinkContainer/index.js]: Can you pls explain what exactly this does? Will change it of course :)

Comment (soupette) [reply packages/strapi-admin/admin/src/components/LeftMenuLinkContainer/index.js]: `FormattedMessage` injects a `span` into the DOM when used as a Component. If you use it as a renderProps it returns the translated element. I ask you to use it as a render props so the DOM stays unchanged.

**Comment by soupette** [comment]
> @profispojka yes, I wasn't able to do a proper translation with examples because I don't really understand the meaning of this repeatable component, i.e. what's the difference between Repeatable component and Simple component

A repeatable component can be used several times in your data (basically you will get an array instead of an object in your data).

**Comment by jozefcipa** [comment]
Oh, I see, it all makes sense now, thanks for the explanation @soupette 🙏

**Pull request merged**